### DISCUSSION TITLE
Do the accounts created by bot get blocked from google?

### DISCUSSION THREAD

**Discussion by ai-to-ai** [leoshabit/Auto-Gmail-Creator#5] [Q&A] [answered]
Hi eveyrone.

Did you check if the gmail is blocked from google after bulk auto creation?

Please let me know 

Thanks

**Comment by adam-eques** [comment, marked as answer]
Hi @leostech 

I've found that the gmail was blocked after auto creation.

Maybe it is related with the gmail numbers which was created on the specific IP address.

**Reply by ai-to-ai** [reply]
Thanks for your announcement
